### PR TITLE
Redesign

### PR DESCRIPTION
**Pull request by leexgh**
image

### CODE REVIEW

File: src/component/variantPage/PopulationPrevalence.tsx [hunk 1]
```diff
@@ -108,50 +110,72 @@ class PopulationPrevalence extends React.Component<IPopulationPrevalenceProps> {
...
             this.props.myVariantInfo &&
             this.props.myVariantInfo.dbsnp &&
             this.props.myVariantInfo.dbsnp.rsid
         ) {
             return (
-                <span className={functionalGroupsStyle['data-group-gap']}>
-                    {this.dbsnpToolTip(
-                        dbsnpUrl,
-                        <span className={functionalGroupsStyle['dbsnp']}>
+                <span className={functionalGroupsStyle['functional-group']}>
```
Comment (alisman) [inline src/component/variantPage/PopulationPrevalence.tsx]: we should not have divs inside spans.  lets just make this a div

File: src/component/variantPage/PopulationPrevalence.tsx [hunk 2]
```diff
@@ -173,16 +197,13 @@ class PopulationPrevalence extends React.Component<IPopulationPrevalenceProps> {
                     </span>
                 }
             >
-                <span className={functionalGroupsStyle['link']}>
+                <span>
```
Comment (alisman) [inline src/component/variantPage/PopulationPrevalence.tsx]: do we need the span?

File: src/component/variantPage/functionalPrediction/MutationAssesor.tsx [hunk 3]
```diff
@@ -277,12 +275,11 @@ export default class MutationAssessor extends React.Component<
...
 
     public render() {
-        let maContent: JSX.Element = (
-            <span className={`${annotationStyles['annotation-item-text']}`} />
-        );
+        let maContent: JSX.Element = <span />;
         const dataSource = (
-            <span className={functionalGroupsStyle['data-source']}>
-                Mutation Assessor
+            <span>
```
Comment (alisman) [inline src/component/variantPage/functionalPrediction/MutationAssesor.tsx]: if this is just text and <span> has no purpose but to contain, we could use React fragment instead.  so, `<>...</> `

**Comment by alisman** [comment]
@leexgh we should try not to put divs inside of spans.  or any block element inside an inline element.

**Comment by leexgh** [comment]
@alisman thanks, updated

**Review by alisman — APPROVED** [review]

**Pull request merged**